sample-apps/SampleAppSpec.md

```diff
@@ -99,7 +99,7 @@ These are Key Value pairs to be added on metrics.
 ```
 {
 “signal”:     (string)“metric”
-“language”:   (string)<name of language used>
+“language”:   (string)<name of language used. Should be set to the name of the sample app preceeding "-sample-app" for standardization purposes>
```

Comment:
Does it make sense to append `-sample-app` to the language attribute? To me the `language` attribute should just reflect the name of the language. Should we prefer `java` over `java-sample-app`?

Reply:
Oh I think I have have worded it incorrectly or not clear enough but I meant exactly that.  So for the java-sample-app, the language should be set to `java` as it is the name of the sample app that precedes `-sample-app`.